package.json

```diff
@@ -67,7 +68,8 @@
     "babel-jest": "27.0.6",
     "babel-loader": "^8.2.5",
     "concurrently": "7.2.2",
-    "cypress": "9.7.0",
+    "cypress": "^10.9.0",
+    "cypress-fail-fast": "^5.0.1",
```

Comment:
Cypress to 10, and added a fast fail library so we save some cycles on GitHub actions. 😎